libsolidity/ast/AST.h

```diff
@@ -345,6 +345,31 @@ class VariableScope
 	std::vector<VariableDeclaration const*> m_localVariables;
 };
 
+/**
+ * The doxygen-style, structured documentation class that represents an AST node,
```

Comment:
Maybe comment that the "structured documentation" is just stored as a single string and not further split up as part of the AST.

Reply:
Or will that change soon?

Reply:
That will change soon, but makes sense for now :)

Reply:
Maybe I'll not add that comment, since it's changed very soon anyways.